Don't update .po files during build.
--- a/gdist/po.py
+++ b/gdist/po.py
@@ -53,10 +53,6 @@
                 os.chdir(self.po_directory)
                 self.spawn(["intltool-update", "--pot",
                             "--gettext-package", self.po_package])
-                for po in self.po_files:
-                    self.spawn(["intltool-update", "--dist",
-                                "--gettext-package", self.po_package,
-                                os.path.basename(po[:-3])])
                 os.chdir(oldpath)
         for po in self.po_files:
             language = os.path.basename(po).split(".")[0]
